superset/assets/javascripts/SqlLab/components/QueryTable.jsx

```diff
@@ -123,7 +123,7 @@ class QueryTable extends React.PureComponent {
           />
         );
       } else {
-        q.output = q.tempTable;
+        q.output = [q.schema, q.tempTable].filter((v) => (v)).join('.');
```

Comment:
sometimes schema may be empty.
Option could be -  get a full name from backend  or just have an if case here